graylog2-web-interface/src/views/components/WidgetContainer.tsx

```diff
@@ -63,7 +62,7 @@ WidgetContainer.defaultProps = {
 };
 
 WidgetContainer.propTypes = {
-  children: PropTypes.element.isRequired,
+  children: PropTypes.any.isRequired,
```

Comment:
I tried finding a better alternative like `PropTypes.node.isRequired` , but in combination with `forwardRef`, I always got a TS error:

```
Type  Validator<NonNullable<ReactNodeLike>>  is not assignable to type  Validator<ReactNode> 
Type  NonNullable<ReactNodeLike>  is not assignable to type  ReactNode 
Type  {}  is not assignable to type  ReactNode 
```